tingbot/platform_specific/osx.py

```diff
@@ -136,6 +136,9 @@ def buttons(self):
 window_controller = None
 
 def fixup_window():
```

Comment:
Now that this function is returning the pygame surface, can we rename `fixup_window` to `create_main_surface`?